salsah/src/public/vendor/ckeditor-4.7.1-Editor/config.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md or http://ckeditor.com/license
+ */
+
+CKEDITOR.editorConfig = function( config ) {
```

Comment:
when you pass a config in, so, in our case when we call:
```javascript
				// init editor (textarea will be replaced by an iframe)
				localdata.editor = CKEDITOR.replace(textarea[0], config);
```

config goes through `CKEDITOR.editorConfig.(config)` and risks to be overridden.